stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,433 @@
+                <label class="fd-rating-indicator__label" for="rating-display-mode-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-3" name="rating-display-mode" value="3" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-4" name="rating-display-mode" value=4" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-5" name="rating-display-mode" value="5" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-5"></label>
+            </div>
+            <span class="dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+    
+    <div class="example-container">
+        <span>Disabled:</span>
+        <div class="fd-rating-indicator" aria-disabled="true">
+            <div class="fd-rating-indicator__container" aria-label="Rating of this item is 2 out of 5">
+                <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-disabled-1" name="rating-disabled" value="1" disabled>  
+                <label class="fd-rating-indicator__label" for="rating-disabled-1"></label>
+                
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-disabled-2" name="rating-disabled" value="2" disabled checked>
+                <label class="fd-rating-indicator__label" for="rating-disabled-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-disabled-3" name="rating-disabled" value="3" disabled>
+                <label class="fd-rating-indicator__label" for="rating-disabled-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-disabled-4" name="rating-disabled" value=4" disabled>
+                <label class="fd-rating-indicator__label" for="rating-disabled-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-disabled-5" name="rating-disabled" value="5" disabled>
+                <label class="fd-rating-indicator__label" for="rating-disabled-5"></label>
+            </div>
+            <span class="dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+`;
+
+States.parameters = {
+    docs: {
+        storyDescription: `These Rating indicator types indicate two states: read-only and disabled.
```

Comment:
Questions about this one:

1. It looks like there are 4 states in the example, so this can be confusing when you say there are two states. Which ones are read-only and which ones are disabled?

2. For the disabled and read-only instructions, when you say for instance: "the disabled attribute needs to be added to the..." are you referring to this `aria-disabled="true"`? Or are there 3 ways to disable the rating indicator?

Suggestion: Add headers (###) for **Disabled** and **Read-only** above each of these paragraphs (instructions). It'll be easier for the user to scan.

Reply:
fixed